Frontend/src/components/TableEditors/Items/Items.svelte

```diff
@@ -4,10 +4,16 @@
   import columns from "./Columns.js";
   import filters from "./Filters.js";
   import { itemDb } from "../../../utils/stores";
+
+  const rowBackgroundColorFunction = (item) => {
+    return item.highlight ?? "";
+  };
 </script>
 
 <DatabaseTableEditor
   {columns}
   {filters}
+  {rowBackgroundColorFunction}
```

Comment:
```suggestion
  {(item) => item.highlight ?? ""}
```

Reply:
this gives an error when I enter it?

Reply:
Sorry, `rowBackgroundColorFunction={(item) => item.highlight ?? ""}`